micro_dl/plotting/plot_utils.py

```diff
@@ -32,12 +32,12 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
     batch_size = len(input_batch)
     if batch_size == 1:
         assert output_fname is not None, 'need fname for saving image'
-        fname = os.path.join(output_dir, '{}.jpg'.format(output_fname))
+        fname = os.path.join(output_dir, '{}.{}'.format(output_fname, ext))
```

Comment:
👍

Reply:
I saw that the preprocessing config switched to 'image_format', 'data_format' is still used in the network config. Here the input image format has to match the the network input format. Should we use 'image_format' for both preprocessing and network configs?